projects/driver_controls/test/test_can_output.c

```diff
@@ -0,0 +1,315 @@
+// Test that the CAN output routines are properly generated once the correct events occur
+
+// Since there isn't a way to view CAN event data directly, the printf statements must be observed
```

Comment:
:|

Reply:
This was a comment from earlier that I didn't remove. I'll get rid of it.